wordpressorg.dev/provision/wp-config.php

```diff
@@ -10,6 +10,8 @@
 define( 'API_WPORGPATH',          __DIR__ . '/../../api.wordpress.org/public_html/includes/' );
 define( 'GLOTPRESS_LOCALES_PATH', __DIR__ . '/../../translate.wordpressorg.dev/public_html/glotpress/locales/locales.php' );
 define( 'WPORG_SANDBOXED',        true );
+define( 'PLUGINS_TABLE_PREFIX',   $table_prefix );
+define( 'WP_CORE_STABLE_BRANCH',  '4.5' );
```

Comment:
This is already defined at https://github.com/WordPress/meta-environment/blob/8c3f7a67a85c0dd4a733083b69cb54159b134141/wordpressorg.dev/provision/sandbox-functionality.php#L34, so it'd probably be better to move that to the top of the file so that all worg.dev sites can use it.